packages/langchain/src/openai/util.test.ts

```diff
@@ -48,8 +48,8 @@ describe('Mapping Functions', () => {
     const defaultOptions = { signal: undefined, promptIndex: 0 };
     const mapping = mapLangchainToAiClient(
       client,
-      defaultOptions,
-      langchainPrompt
+      langchainPrompt,
+      defaultOptions
```

Comment:
[pp] If you have default options, shouldn't those be considered in `mapLangchainToAiClient()` by **default** without passing them?

Reply:
The reason I pass them here is to replicate the actual behavior when calling the client.
The super class always passes these options.